roles/compute/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+#
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
```

Comment:
copyrights for new files and any changed files should be 2023 (or XXXX-2023 for existing files if any)

Reply:
Updated.